### ISSUE TITLE
Scan Multiple Pages

### ISSUE THREAD

**Issue by NaitikShah** [ashishkataria/browserWebScanning#2]
Hey Ashish Please help with this...
I am very thankful to have your project but I also want to scan Multiple Pages and also convert them in various formats.
If u can please share your knowledge and help me further then it would be a great help for me.
Thanks in Advance.
~Naitik

**Comment by ashishkataria** [comment]
Thanks Naitik, Presently scanning multiple pages feature is not available. 
This will definitely extend its capabilities to  achieve full fledge application status. I appreciate your idea.
We can keep on storing a copy of each scanned image somewhere into java  byteArray objects  in loops till all scanning is done, and later on can display all pages giving conversion options. This is just a brief idea.
Infact, I would be happy to help.
you can write to me at ashishkatarias@gmail.com
Thanks.

**Comment by NaitikShah** [comment]
Yes i am trying the same since a week...but i am thankful to your code that gave me a start atleast.
